### ISSUE TITLE
Copy DisableRetryOnRateLimit

### ISSUE THREAD

**Issue by LipatovAlexander** [soxtoby/SlackNet#155]
In the #152 , you suggested such a way to disable SlackNet retry on rate limit:
```csharp
((SlackNet.SlackApiClient)slackApiClient).DisableRetryOnRateLimit = true;
```
I did this, but I have to repeat this on every call `.WithAccessToken(...)`. Perhaps it would be more convenient if this property were copied when creating a new client.

```csharp
public ISlackApiClient WithAccessToken(string accessToken) => new SlackApiClient(_http, _urlBuilder, _jsonSettings, accessToken)
{
  DisableRetryOnRateLimit = DisableRetryOnRateLimit
};
```

**Comment by soxtoby** [comment]
Yep, this is definitely just an oversight on my part. I'll push up a fix shortly.

**Comment by soxtoby** [comment]
Fix is now available in [v0.10.23](https://github.com/soxtoby/SlackNet/releases/tag/v0.10.23).
